lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -876,6 +876,9 @@ llvm::Error DAP::Loop() {
     }
   }
 
+  if (output.descriptor)
+    output.descriptor->Close();
```

Comment:
should this be closed in the cleanup block above?

Thanks for the quick fix!

Reply:
Done.